libs/ledger-live-common/src/featureFlags/defaultFeatures.ts

```diff
@@ -499,6 +499,10 @@ export const DEFAULT_FEATURES: Features = {
   spamFilteringTx: DEFAULT_FEATURE,
   llmMemoTag: DEFAULT_FEATURE,
   lldMemoTag: DEFAULT_FEATURE,
+  ldmkTransport: {
+    ...DEFAULT_FEATURE,
+    params: { stale: true },
```

Comment:
This param isn't used right? otherwise it should have a type

Reply:
@jdabbech-ledger leftover from prev. implementation! thank you